clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/domain/LoadBalancerPool.groovy

```diff
@@ -41,7 +42,22 @@ class LoadBalancerPool implements LoadBalancerResolver {
 
   void setInternalPort(Integer port) {
     this.internalPort = port
-    this.description = "internal_port=${internalPort}"
+    String portString = generateInternalPort(port)
+    if (this.description) {
+      this.description += ",$portString"
+    } else {
+      this.description = portString
+    }
+  }
+
+  void setCreatedTime(long time) {
```

Comment:
If we need to keep adding things to the description, I would recommend a method for appending to the description. With just two methods, I am not worried yet.